beacon-chain/execution/engine_client.go

```diff
@@ -30,6 +30,20 @@ import (
 	"go.opencensus.io/trace"
 )
 
+var (
```

Comment:
I guess we should also include `ExchangeCapabilities` itself.

Reply:
In the spec it says "The engine_exchangeCapabilities method MUST NOT be returned in the response list.". So I removed it from the list of our own supported ones as well. 
Would it be better to include it?